qlib/backtest/exchange.py

```diff
@@ -714,6 +713,63 @@ def _clip_amount_by_volume(self, order: Order, dealt_order_amount: dict) -> int:
                 f"Order clipped due to volume limitation: {order}, {[(vol, rule) for vol, rule in zip(vol_limit_num, vol_limit)]}"
             )
 
+    def _cal_trade_amount_by_cash_limit(self, now_trade_amount, trade_price, order, position):
+        """return the real order amount after cash limit.
+
+        Parameters
+        ----------
+        now_trade_amount : float
+        trade_price : float
+        order : Order
+        position : Position
+
+        Return
+        ----------
+        float
+            the real order amount after cash limit.
+        """
+        cash = position.get_cash()
+        trade_val = now_trade_amount * trade_price
+        if order.direction == Order.SELL:
+            if cash < trade_val * self.close_cost:
```

Comment:
Let's have some discussions about this value.

I think we must add some tests for this case